react-app/src/api/bankAPI.ts

```diff
@@ -31,7 +31,7 @@ export const useBankAPI = (): IBankAPI => {
       const balance = await cosmos.getBalance();
       const coinBalance = convertTokenToMinimalToken(balance.amount);
 
-      if (coinBalance.isLessThan(coinAmount)) {
+      if (coinBalance.isLessThanOrEqualTo(coinAmount)) {
```

Comment:
but how about gas fee...
& allow use up all funds should be throw at balance < amount not <= amount? now it does not allow txn for that condition?

Reply:
oh you right